cli_tools/common/image/importer/bootable_disk_processor.go

```diff
@@ -87,6 +92,25 @@ func newBootableDiskProcessor(request ImageImportRequest, wfPath string, logger
 	return diskProcessor
 }
 
+// This function is to attach the created data disks into the translation worker instance, to handle if part
+// of the OS data (e.g. /var) is mounted onto one of the data disks
+// currently it is supported for Linux only.
+func updateWorkflowWithDataDisks(wfPath string, wf *daisy.Workflow, request ImageImportRequest) {
+	if !strings.Contains(strings.ToLower(wfPath), "windows") {
+		var disks *[]*compute.AttachedDisk
+		if wf.Steps["translate-disk"] != nil {
+			disks = &wf.Steps["translate-disk"].IncludeWorkflow.Workflow.Steps["translate-disk-inst"].CreateInstances.Instances[0].Disks
+		} else if wf.Steps["translate-disk-inst"] != nil {
+			// case translation wf that doesn't include another workflow (e.g. opensuse_15)
+			disks = &wf.Steps["translate-disk-inst"].CreateInstances.Instances[0].Disks
+		}
+
+		for _, dataDisk := range request.DataDisks {
+			*disks = append(*disks, &compute.AttachedDisk{Source: dataDisk.GetURI()})
+		}
```

Comment:
I'd suggest a follow-up PR that traverses the workflow as a tree and looks for `CreateInstances` nodes, rather than hardcoding the step names. I think that'll be more robust, since this code assumes naming conventions, which would cause the traversal to break.